Snakefile

```diff
@@ -143,7 +143,7 @@ rule prepare_re_potential:
         pv_road_railway="raw/area_potential/2021-05-18_pv_road_railway_brandenburg_kreise_epsg32633.csv",
         wind="raw/area_potential/2021-05-18_wind_brandenburg_kreise_epsg32633.csv",
         kreise="raw/lookup_table_brandenburg_kreise.csv",
-        assumptions="raw/scalars.csv",
+        assumptions="raw/scalars_base.csv",
         script="scripts/prepare_re_potential.py"
     output:
         directory("results/_resources/RE_potential/")
```

Comment:
I don't know if this is already planned / intended here, but this would ensure that the correct scalars are selected via wildcards:

```suggestion
        assumptions="raw/scalars_{range}.csv",
        script="scripts/prepare_re_potential.py"
    output:
        directory("results/_resources/{range}/RE_potential/")
    wildcard_constraints:
        range=("base|high|low")
```

 However, if we were to build on our current scalars structure, then the wildcards would have to be extended by the specification of a year and hence adjusted to:

```suggestion
        assumptions="raw/scalars_{range}_{year}.csv",
        script="scripts/prepare_re_potential.py"
    output:
        directory("results/_resources/{range}_{year}/RE_potential/")
    wildcard_constraints:
        range=("base|high|low"),
        year=("2040|2050")        
```

Just a thought without any particular change request.

Reply:
Thank you @MaGering! 
The thing is, that using the different csvs does not have any effect on the input data to the simulations, as the renewable potential is not copied to `_resources/scal_base.csv`. 
I would add the potential automatically in `prepare_scalars.py` using `results/_resources/scal_power_potential_wind_pv.csv` as input. Do you agree @jnnr?

Reply:
@MaGering I will include that, giving you credit, in #176 after the release. thanks again!